spec/support/authentication_helper.rb

```diff
@@ -8,4 +8,9 @@ def sign_in_as(user)
 
     click_on 'Log in'
   end
+
+  def sign_out_as(user)
```

Comment:
Unused method argument - `user`. If it's necessary, use `_` or `_user` as an argument name to indicate that it won't be used. You can also write as `sign_out_as(*)` if you want the method to accept any arguments but don't care about them.